### ISSUE TITLE
Healthy samples only have CD45+ cells, will it affect subsequent analysis such as drawing cell ratio maps?

### ISSUE THREAD

**Issue by realmanlxw** [zhangzlab/covid_balf#22]
Excellent work. I checked your raw data in the GEO database and found that healthy samples were sequenced CD45+ cells, while other samples were total cells. Will this affect the drawing of cell ratio maps and subsequent analysis? Looking forward to your reply

**Comment by polojacky** [comment]
You are right, the healthy samples from our lab only have CD45+ cells. So if you want to compare cell ratio, you should only do this in CD45+ cell groups.

**Comment by realmanlxw** [comment]
After observing your follow-up work, do you think that except epithelial cells, other immune cell types are CD45+ cells?

**Comment by polojacky** [comment]
> After observing your follow-up work, do you think that except epithelial cells, other immune cell types are CD45+ cells?

Yes, CD45 is a commonly used marker for immune cells selection.

**Comment by realmanlxw** [comment]
Thanks again for your reply
